setup.py

```diff
@@ -171,7 +179,7 @@ def write_version(filename=os.path.join(*['airflow',
 doc = [
     'sphinx-argparse>=0.1.13',
     'sphinx-autoapi>=0.7.1',
-    'Sphinx-PyPI-upload>=0.2.1',
+    'Sphinx-PyPI-upload3>=0.2.2',
```

Comment:
We could remove this I think - we don't use it as part of the release process - I don't even quite know what asset it uploads/how pypi might display it if we did.

Reply:
Yeah lets remove this @BasPH

Reply:
Removed the dependency + related config in setup.cfg.